GameAI/Navigation/DefaultPathSmoother.cs

```diff
@@ -9,6 +9,6 @@ public class DefaultPathSmoother : IPathSmoother
         /// Does not smooth paths
         /// </summary>
         /// <inheritdoc cref="IPathSmoother#SmoothPath"/>
-        public IEnumerable<Vector2> SmoothPath(IEnumerable<Vector2> path) => path;
+        public IEnumerable<Vector2> SmoothPath(LinkedList<Vector2> path) => path;
```

Comment:
Waarom? Is het niet beter om binnen de CustomizablePathSmoother een LinkedList aan te maken?